packages/react/src/auto/hooks/useStringInputController.tsx

```diff
@@ -32,7 +32,6 @@ export const useStringInputController = (props: {
     fieldState: { error },
   } = useController({
     name: path,
-    rules: { required: metadata.requiredArgumentForInput },
```

Comment:
The existing "required" validation is actually done in the `validationSchema.ts` file, so I removed places that make this custom rule to prevent confusion.